src/Wordpress/CustomPostType/CustomPostType.php

```diff
@@ -43,13 +45,24 @@ public static function getWPNonceId(): string {
 
 	/**
 	 * Replaces WP_Posts by their title for options array.
+	 * If $allOption is true, an additional option is added to the beginning of the array that allows to select all posts
+	 * This option will only be shown to the Administrator role, since only Administrators can usually see all posts
+	 *
+	 * TODO:
+	 *  - All could be interpreted differently depending on the context of the editor. So in the case of a CB Manager "All" could mean all items or locations they manage.
 	 *
-	 * @param $data
+	 * @param WP_Post[]|string[] $data
 	 *
 	 * @return array
 	 */
-	public static function sanitizeOptions( $data ) {
+	public static function sanitizeOptions( $data, $allOption = false ) {
 		$options = [];
+
+		// Add option to select all items
+		if ( $allOption && current_user_can( 'administrator' ) ) {
+			$options[self::SELECTION_ALL_POSTS] = __( 'All', 'commonsbooking' );
```

Comment:
An der Stelle könnte man doch auch eine ALLE Option einfügen für die Manager-Rolle, sowas wie `All currently managed by you`, dann spart man sich das TODO oben

Reply:
Bzw. ich verstehe den Code so das es aktuell nur Administrator-Rollen möglich ist die `All` Option zu sehen. Das heißt der Fall im Todo ist doch gar nicht möglich, da Manager-Rollen `All` nie sehen würden.

Reply:
Na ja, theoretisch müsste das ja nicht so sein. Je nachdem welcher post_author gesetzt ist könnte "all" im verschiedenen Kontext was anderes bedeuten. Wenn der post_author zb CB Manager ist und zwei Items zugewiesen hat, wäre all für den post_author diese beiden Items.